infra/app-configuration/configuration-store.bicep

```diff
@@ -0,0 +1,64 @@
+@description('Required. The parameter object for the virtual network. The object must contain the name,resourceGroup and subnetPrivateEndpoints values.')
+param vnet object
+
+@description('Required. The parameter object for appConfig. The object must contain the name, privateEndpointName, softDeleteRetentionInDays and enablePurgeProtection values.')
+param appConfig object
+
+@allowed([
+  'Free'
+  'Standard'
+])
+@description('Optional. Pricing tier of App Configuration.')
+param sku string = 'Standard'
+
+@description('Optional. The Azure region where the resources will be deployed.')
+param location string = resourceGroup().location
+
+@description('Required. Environment name.')
+param environment string
+
+@description('Required. Date in the format yyyyMMdd-HHmmss.')
+param deploymentDate string = utcNow('yyyyMMdd-HHmmss')
+
+@description('Required. Date in the format yyyy-MM-dd.')
+param createdDate string = utcNow('yyyy-MM-dd')
+
+var customTags = {
+  Location: location
+  CreatedDate: createdDate
```

Comment:
I think this should be changed to CreatedOrUpdatedDate in all the modules

Reply:
@kbabigumira - Updated to CreatedOrUpdatedDate in all modules.

Reply:
Dan clarified that this should remain as CreatedDate. There is a requirement to have a CreatedDate Tag for resources. It would require a script to conditionally set the CreatedDate but there is little benefit in spending effort on a script to do this.